src/fairchem/core/common/utils.py

```diff
@@ -779,6 +780,114 @@ def radius_graph_pbc(
     return edge_index, unit_cell, num_neighbors_image
 
 
+def generate_graph_standalone(
```

Comment:
can we just call it `generate_graph`? or is there a name clash somewhere that I missed?

Reply:
Agree and fixed!